### ISSUE TITLE
[Android]: A crash in com.mapswithme.maps.MwmApplication.prefs(MwmApplication.java:134)

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#205]
```
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getString(int)' on a null object reference
       at com.mapswithme.maps.MwmApplication.prefs(MwmApplication.java:134)
       at com.mapswithme.maps.location.TrackRecorder.getAwaitTimeout(TrackRecorder.java:168)
       at com.mapswithme.maps.location.TrackRecorderWakeService.onHandleWork(TrackRecorderWakeService.java:48)
       at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:392)
       at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:383)
       at android.os.AsyncTask$3.call(AsyncTask.java:394)
       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:923)
```

https://console.firebase.google.com/u/0/project/omapsapp/crashlytics/app/android:app.omaps.beta/issues/9d128f9041cbaed3f2b4550ea55ba622?time=last-seven-days&sessionEventKey=60452198035F00016F19240AB7636309_1515433346452933548

**Comment by rtsisyk** [comment]
Fix problem should be fixed in 10.6.0.

**Comment by rtsisyk** [comment]
This problem is still actual in 10.6.3.
https://console.firebase.google.com/u/0/project/omapsapp/crashlytics/app/android:app.omaps/issues/a1122122712309848c93886bd1ecdc60?time=last-seven-days&sessionEventKey=6055F256010600013BDD667C4B59109E_1520166456668810841
